x-pack/solutions/observability/plugins/inventory/public/components/entity_group_accordion/index.tsx

```diff
@@ -41,17 +41,17 @@ export function EntityGroupAccordion({ groupBy, groupValue, groupCount, isLoadin
         `}
       >
         <EuiAccordion
-          data-test-subj={`inventoryGroup_${groupBy}_${groupValue}`}
-          id={`inventory-group-${groupBy}-${groupValue}`}
+          data-test-subj={`inventoryGroup_entity.type_${groupValue}`}
+          id={`inventory-group-entity.type-${groupValue}`}
           buttonContent={
             <EuiTitle size="xs">
-              <h4 data-test-subj={`inventoryGroupTitle_${groupBy}_${groupValue}`}>{groupValue}</h4>
+              <h4 data-test-subj={`inventoryGroupTitle_entity.type_${groupValue}`}>{groupLabel}</h4>
```

Comment:
Nit: To follow the name convention should we use `entityType` here so we will have `inventoryGroupTitle_entityType_${groupValue}`? We can also keep it as the `entity.type` is the actual groupBy.